bedrock/firefox/templates/firefox/whatsnew/whatsnew-56.html

```diff
@@ -74,6 +74,11 @@ <h3 class="new-tab">{{ _('Keep it fresh') }}</h3>
         <p>{{ _('The all-new New Tab gives you quick access to your top sites and recommendations from around the web.') }}</p>
       </li>
     </ul>
+
+    {# Bug 1401308: Only link to /firefox/quantum/ for locales that have the page translated. #}
+    {% if LANG.startswith('en') or LANG == 'de' %}
```

Comment:
Will the quantum page only ever be available in English and German? Is there a way to use the `lang_file_is_active` function to decide whether to show this button, rather than hard-code specific locales? Or maybe the quantum page will be short-lived enough not to worry about it.

Reply:
Only ever German & English. When 57 lands it will become /firefox and then get translated.

Reply:
No worries then!